src/index.html

```diff
@@ -11,12 +11,110 @@
       content="ie=edge"
     />
     <title>Moyo header</title>
+    <link
+      rel="preconnect"
+      href="https://fonts.googleapis.com"
+    />
+    <!-- font -->
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+      crossorigin="anonymous"
+    />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap"
+      rel="stylesheet"
+    />
+    <!-- css -->
     <link
       rel="stylesheet"
       href="./style.css"
     />
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <!-- header -->
+    <header class="header">
+      <!-- logo -->
+      <a
+        href="#"
+        class="logo"
+      >
+        <img
+          class="logo-img"
+          src="images/logo.png"
+          alt="logo"
+        />
+      </a>
+      <!-- nav -->
+      <nav class="nav">
+        <ul class="nav-list">
+          <li class="nav-item">
+            <a
+              href="#"
+              class="nav-link is-active"
+            >
+              Apple
+            </a>
+          </li>
+          <li class="nav-item">
+            <a
+              href="#"
+              class="nav-link"
+            >
+              Samsung
+            </a>
+          </li>
+          <li class="nav-item">
+            <a
+              href="#"
+              class="nav-link"
+            >
+              Smartphones
+            </a>
+          </li>
+          <li class="nav-item">
```

Comment:
```suggestion
          <li class="nav-item">
            <a
              href="#"
              class="nav-link"
            >
              Smartphones
            </a>
          </li>
          
          <li class="nav-item">
```

Add an empty line between multiple sibling blocks of html